crates/bevy_render/src/texture/image.rs

```diff
@@ -115,9 +115,8 @@ pub struct Image {
 impl Default for Image {
     fn default() -> Self {
         let format = wgpu::TextureFormat::bevy_default();
-        let data = vec![255; format.pixel_size() as usize];
         Image {
-            data,
+            data: Vec::new(),
```

Comment:
In theory, I'd rather this used an enum.

Reply:
Maybe we should just use `Option<Vec<u8>>` to be clear about whether the data is even accessible on the GPU. @superdump mentioned on discord that they wanted to look into avoid loading data in system ram at all, and instead load directly into vram. This may be relevant here as well.

Reply:
The uses cases are more complex there. I think some more thought is needed to flesh that out and so I don't know that it needs to hold this up. Personally for the scope of this PR, I feel like an Option<> would sufficiently communicate whether the data exists in system RAM or not.